core/domain/action_registry_test.py

```diff
@@ -29,3 +29,12 @@ def test_action_registry(self):
         """Do some sanity checks on the action registry."""
         self.assertEqual(
             len(action_registry.Registry.get_all_actions()), 3)
+
+    def test_get_action_by_type(self):
+        # Testing with invalid action type.
+        with self.assertRaisesRegex(KeyError, 'fakeAction'):
+            action_registry.Registry.get_action_by_type('fakeAction')
+
+        # Testing with valid action type.
```

Comment:
We're testing two different things here (valid and invalid), so you can split this into 2 tests

Reply:
Done!